bom/openhab-core/pom.xml

```diff
@@ -226,6 +226,12 @@
       <version>${project.version}</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.openhab.core.bundles</groupId>
+      <artifactId>org.openhab.core.io.rest.transform</artifactId>
```

Comment:
If you keep the dependencies sorted, it will be easier to find each dependency in the BOM.